product_docs/docs/migration_toolkit/55/07_invoking_mtk.mdx

```diff
@@ -98,7 +92,7 @@ Where:
 
 `-targetdbtype target_type`
 
-`target_type` specifies the server type of the target database. `target_type` is case-insensitive. By default, `target_type` is `enterprisedb`. `target_type` may be one of the following values:
+`target_type` specifies the server type of the target database. `target_type` is not case sensitive. By default, `target_type` is `enterprisedb`. `target_type` can be one of the following values:
```

Comment:
Should we use angular brackets here?

Reply:
Yes, and if you look at the command, it says <dbtype> where it should say <source_type>. I will fix that and add the angle brackets to the explanatory text for that as well.